geniza/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+from geniza.corpus.models import Document, Fragment
+
+class GenizaAdminSite(admin.AdminSite):
+    '''A customized AdminSite to aid PGP workflow'''
+    def each_context(self, request):
+        '''Provide extra context dictionary to admin site'''
+        REVIEW_PREVIEW_MAX = 10
```

Comment:
I'd define this at the level of `GenizaAdminSite` so it doesn't get re-declared on each call to `each_context()` (which we think happens every pageload in the admin).